<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>expeditionmanager</class>
 <widget class="QDialog" name="expeditionmanager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="option4">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>280</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="option2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>280</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="eventLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>1210</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="option1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>280</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="option3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>280</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
